project/BuildSettings.scala

```diff
@@ -109,7 +109,13 @@ object Doc extends AutoPlugin {
 
   lazy val scalaDocSettings = Seq(
     autoAPIMappings := true,
-    apiURL := Some(url(s"http://api.silhouette.rocks/${version.value}/"))
+    apiURL := Some(url(s"http://api.silhouette.rocks/${version.value}/")),
+    apiMappings ++= {
```

Comment:
The `apiMappings` definition is no longer needed because of https://github.com/minutemen/silhouette/blob/master/project/plugins.sbt#L23